istio/istio.go

```diff
@@ -437,6 +465,37 @@ func (iClient *IstioClient) ApplyOperation(ctx context.Context, arReq *meshes.Ap
 		return &meshes.ApplyRuleResponse{
 			OperationId: arReq.OperationId,
 		}, nil
+	case googleMSSampleApplication:
+        go func() {
+            opName1 := "deploying"
+            if arReq.DeleteOp {
+                opName1 = "removing"
+            }
+            if err := iClient.executeHipsterShopInstall(ctx, arReq); err != nil {
+                iClient.eventChan <- &meshes.EventsResponse{
+                    OperationId: arReq.OperationId,
+                    EventType:   meshes.EventType_ERROR,
+                    Summary:     fmt.Sprintf("Error while %s thegoogle micro services demo applicatio", opName1),
```

Comment:
Please add a space between "the" and "google", and a "n" to the end of application.

Reply:
Can you please add a space between "the" and "google", and a "n" to the end of application?